modules/util/portal-tools-sample-sql-builder/src/main/java/com/liferay/portal/tools/sample/sql/builder/DataFactory.java

```diff
@@ -9128,6 +9139,7 @@ private List<String> _readLines(String resourceName) throws Exception {
 
 	private static final Log _log = LogFactoryUtil.getLog(DataFactory.class);
 
+	private static final Pattern _pattern = Pattern.compile("liferay(.*)");
```

Comment:
@slnn Why you have a hardcoded pattern here? what's this for?

Reply:
@slnn Can you explore to directly create the web IDs of the non-admin companies with the prefix, so that we don't need to use a regexp later?

Reply:
> @slnn Why you have a hardcoded pattern here? what's this for?

Hi @tinatian 

The pattern is used to get the number of  the non-admin companies' webId, for example, the weId is  liferay1.com, the pattern aims to get the number "1", and SF requires to make it as a global variable when using the pattern.

Reply:
> @slnn Can you explore to directly create the web IDs of the non-admin companies with the prefix, so that we don't need to use a regexp later?

OK

Reply:
Done